plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -70,6 +71,7 @@ func NewSchedulerServer() *SchedulerServer {
 		BindPodsBurst:     100,
 		KubeAPIQPS:        50.0,
 		KubeAPIBurst:      100,
+		SchedulerName:     "kube-scheduler",
```

Comment:
I would suggest to call it "default-scheduler" everywhere--since we call the scheduler binary kube-scheduler, this will avoid confusion when people "grep" for one or the other.